css/work.css

```diff
@@ -0,0 +1,166 @@
+.work {
```

Comment:
- [ ] [OPTIONAL] Kindly check this console error in the browser, the problem seems to be the inability to load images from this file. 
![image](https://user-images.githubusercontent.com/61408860/167455656-82275d0e-ae34-4706-8414-2dc4eac6c222.png)

Reply:
I have fixed those issues